File: Chapter09/Java/PreferencesDemo/app/src/main/java/com/androidbook/preferences/sample/FlightPreferenceFragment.java
package com.androidbook.preferences.sample;

import android.os.Bundle;

import androidx.preference.PreferenceFragmentCompat;

public class FlightPreferenceFragment extends PreferenceFragmentCompat {

    @Override
    public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
        setPreferencesFromResource(R.xml.flightoptions, rootKey);
    }
}

File: Chapter09/Kotlin/PreferencesDemo/app/src/main/java/com/androidbook/preferences/sample/FlightPreferenceFragment.kt
package com.androidbook.preferences.sample

import android.os.Bundle
import androidx.preference.PreferenceFragmentCompat

class FlightPreferenceFragment : PreferenceFragmentCompat() {

    override fun onCreatePreferences(savedInstanceState: Bundle?, rootKey: String?) {
        setPreferencesFromResource(R.xml.flightoptions, rootKey)
    }
}